proto/tabletmanagerdata.proto

```diff
@@ -545,6 +545,7 @@ message ReplicaWasRestartedResponse {
 
 message StopReplicationAndGetStatusRequest {
   replicationdata.StopReplicationMode stop_replication_mode = 1;
+  replicationdata.Capability capability = 2;
```

Comment:
We pass the "capability" to the RPC because we already know it at this stage

We **_could_** have the receiving tablet determine this instead, but that will require loading the durability policy in the `grpctmserver`, which will cause a N x `vttablet` fanned-out topo read for the shard record _(that contains the durability policy_). This approach avoids that 👍